cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -416,16 +416,20 @@ bool ProcessShowQASubActionAdd(const std::multimap<std::string, std::string> &cg
     // sub-action "add"
     const std::string add_type(GetCGIParameterOrDefault(cgi_args, "add_type"));
     const std::string add_tag(GetCGIParameterOrDefault(cgi_args, "add_tag"));
+    const std::string add_subfield_code(GetCGIParameterOrDefault(cgi_args, "add_subfield_code"));
```

Comment:
What about cpp/data/zotero_cgi/qa.html? (file is missing in the Pull Request so far)